apps/judicial-system/web/src/components/AccordionItems/CommentsAccordionItem/CommentsAccordionItem.tsx

```diff
@@ -45,7 +45,7 @@ const CommentsAccordionItem: React.FC<Props> = (props) => {
             </Text>
           </Box>
           <Text whiteSpace="breakSpaces">
-            <MarkdownWrapper text={workingCase.caseResentExplanation} />
```

Comment:
Here is another example of our build failing. This should not compile. There is no longer a prop called `text` in `MarkdownWrapper`

Reply:
The correct prop is `markdown`.